src/Database/Database.php

```diff
@@ -135,7 +136,11 @@ public function connect()
 				$connect .= ";charset=" . $charset;
 			}
 
-			try {
+		if ($socket) {
+			$connect .= ";$unix_socket=" . $socket;
+		}
+
+		try {
```

Comment:
Code standards: Please normalize the indentation level.
```suggestion
			if ($socket) {
				$connect .= ";$unix_socket=" . $socket;
			}

			try {
```